exercises/zebra-puzzle/canonical-data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "exercise": "zebra-puzzle",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "resident who drinks water",
+      "property": "nationality",
```

Comment:
Should the properties be: 'drinks_water' and 'owns_zebra' ?

Reply:
I was thinking that this would be the return "type" of the expected value. If we do adjust the exercise to use different clues, then this would be a good way to know what data "type" the exercise would be testing for. Or would this not be a good use of the `"property"` tag? We could alternatively use `"nationality"`, etc. as the key instead of `"expected"`.

Reply:
But we're always asking for nationality. How does the test generator know whose nationality we want?

If the exercise changes, the test data will also need to change so I would not worry about future proofing at this stage.

Reply:
Oh I understand now. And yes, good point. Changing now.